src/simulator_core/entities/assets/demand_cluster.py

```diff
@@ -103,12 +101,10 @@ def create(self) -> None:
             )
             # Create the control valve
             self._flow_control = ControlValve(
-                pandapipes_net=self.pandapipes_net,
-                controlled_mdot_kg_per_s=self.mass_flowrate,
-                diameter_m=self._internal_diameter,
-                control_active=True,
-                in_service=True,
-                name=f"flow_control_{self.name}",
+                pandapipe_net=self.pandapipes_net,
+
```

Comment:
Er zit hier nog een enter. Waarom zet je ook niet meer de parameters? Haal je die nu de ControlValve class? 

Zo ja dan is het misschien ook een idee om de self._internal_diameter te verwijderen uit de andere aanroepen en het op dezelfde manier op te lossen?

Reply:
Adjusted to set the default diameter when diameter is not added.